pkgs/top-level/all-packages.nix

```diff
@@ -2557,7 +2557,9 @@ with pkgs;
 
   gopass-summon-provider = callPackage ../tools/security/gopass/summon.nix { };
 
-  gosca = callPackage ../development/tools/gosca { };
+  gosca = callPackage ../development/tools/gosca {
```

Comment:
https://github.com/sjas/gosca gosca is also deprecated, archived, also possible candidate for deletion. I'm mostly suggesting deleting packages that won't be made to work upstream with 1.18 because 1.19 is around the corner, at which point 1.17 will no longer be supported. Keeping packages that can't be upgraded (or don't make sense to be upgraded) to work with 1.18 at that point will not be that good.